agent_build/docker/Dockerfile

```diff
@@ -39,8 +39,18 @@ RUN tar -xf /tmp/build/scalyr-agent.tar.gz
 
 WORKDIR /
 
+FROM python:3.8.16-slim-bulseye as scalyr-base-slim-bullseye
+MAINTAINER Scalyr Inc <support@scalyr.com>
+RUN apt-get update ; apt-get dist-upgrade -y
```

Comment:
Please add a comment why we do this (to ensure the base image is up to date with all security updates, etc.).